recordium/main.py

```diff
@@ -17,16 +17,28 @@
 import logging
 import subprocess
 import sys
+import platform
+import os
+import PyQt5
 
 from functools import lru_cache
 
 from PyQt5 import QtWidgets, QtGui, QtCore
+from PyQt5 import __file__ as pyqt_path
 
 from recordium import network, storage
 from recordium.config import config
 
 logger = logging.getLogger(__name__)
 
+def fix_environment():
+    '''add enviroment variable on Windows systems'''
```

Comment:
The rules for docstrings say this should be: ` """Add enviroment variable on Windows systems."""`